src/Map.jsx

```diff
@@ -75,7 +75,14 @@ class Map extends Component {
       { force: true },
     );
   }
-
+  getStopsWithSid(route) {
+    const stops = this.state.selectedStops;
+    const indexArray = [];
+    indexArray.push(route.stops.findIndex(stop => stop.sid === stops[0].sid));
+    indexArray.push(route.stops.findIndex(stop => stop.sid === stops[1].sid));
```

Comment:
Hey, so it seems like you're storing the stop's index in the list of stops instead of the `stop_id`? I think the latter would make more sense as then we're not making unnecessary `find` calls.